crates/uv-trampoline/Cargo.toml

```diff
@@ -6,46 +6,41 @@ license = "MIT OR Apache-2.0"
 edition = "2021"
 autotests = false
 
-
 # Need to optimize etc. or else build fails
 [profile.dev]
-lto = true
-codegen-units = 1
-opt-level = 1
 panic = "abort"
-debug-assertions = false
-overflow-checks = false
 debug = true
 
 [profile.release]
-lto = true
-codegen-units = 1
-opt-level = "z"
-panic = "abort"
+lto = true          # Enable Link Time Optimization
+codegen-units = 1   # Reduce number of codegen units to increase optimizations.
+opt-level = "z"     # Optimize for size.
+panic = "abort"     # Abort on panic
+strip = true        # Automatically strip symbols from the binary.
```

Comment:
Nit: Can you remove the block alignment of the comments? Either move the to the previous line or one space after the item they are commenting on